vmdb/app/helpers/application_helper.rb

```diff
@@ -1002,8 +1002,10 @@ def build_toolbar_hide_button(id)
         return true if !@record.is_available?(:shutdown_guest)
       when "vm_guest_restart", "instance_guest_restart"
         return true if !@record.is_available?(:reboot_guest)
-      when "vm_migrate", "vm_reconfigure"
-        return true if @record.vendor.downcase == "redhat"
+      when "vm_migrate"
+        return true unless @record.supports_migrate?
+      when "vm_reconfigure"
+        return true unless @record.supports_reconfigure?
```

Comment:
Actually I just noticed that the lines above already have a mechanism called `is_available?`.  Should this code also be in the `is_available?` code instead of new methods, for consistency?  I have no idea what `is_available?` does.